.travis.yml

```diff
@@ -4,6 +4,13 @@ env:
     - LD_PRELOAD=/lib/x86_64-linux-gnu/libSegFault.so
     - SEGFAULT_SIGNALS=all
 
```

Comment:
Better to simply excise this section than have it exist with nothing but commented-out contents.